docs/specs/composition.md

```diff
@@ -1,12 +1,45 @@
 # Composition
 
-
 <h2 id="animation">Animation</h2>
 
 {schema_string:composition/animation/description}
 
 {schema_object:composition/animation}
 
+### Versioning Guidelines
+
+Tools implementing the Lottie specification SHOULD consider the following
+guidelines:
+
+* Major version updates MAY contain breaking changes that are not compatible
+with previous versions of the specification.
+* Minor version updates are typically adding new functionality and SHOULD NOT
+contain breaking changes.
+* Patch version updates are typically making minor changes to already existing
+functionality.
```

Comment:
Since I'm writing about the specs release process in the contributing file, I'll mention semantic versioning in there too